sandboxed_containers/sandboxed-containers-release-notes.adoc

```diff
@@ -0,0 +1,167 @@
+
+[id="node-eligibility-rn"]
+=== Check node eligibility to run {sandboxed-containers-first}
+
+Administrators can now check the eligibility of cluster nodes to run {sandboxed-containers-first}. This feature uses the Node Feature Discovery (NFD) operator to detect  node capabilities. Eligible nodes are labeled with `feature.node.kubernetes.io/runtime.kata`, and the {sandboxed-containers-operator} uses this label to select candidate nodes for installation.
+
+The administrator must deploy the NFD operator to use this feature, create a specific *NodeFeatureDiscovery* custom resource, and enable `checkNodeEligibility' when creating the *KataConfig* custom resource.
+//For more information, see xref:../sandboxed_containers/deploying-sandboxed-container-workloads.adoc#sandboxed-containers-check-nodes_deploying-sandboxed-containers[Checking the eligibility of cluster nodes to run {sandboxed-containers-first}].
+
+[id="OSC-compatibility-with-CNV"]
+=== {sandboxed-containers-first} compatibility with {VirtProductName}
+
+Users can now run {sandboxed-containers-first} on clusters with {VirtProductName} when VMs are configured correctly.
+For more information, see xref:../sandboxed_containers/understanding-sandboxed-containers.adoc#sandboxed-containers-with-other-ocp-components_understanding-sandboxed-containers[Using {sandboxed-containers-first} with {VirtProductName}]
+
+[id="support-AWS-bare-metal"]
+=== {sandboxed-containers-first} availability on AWS bare-metal (tech preview)
+
+Users can now install {sandboxed-containers-first} on AWS bare-metal clusters. This feature is in tech preview and not fully supported. For more information, see xref:../sandboxed_containers/understanding-sandboxed-containers.adoc#understanding-sandboxed-containers[Understanding {sandboxed-containers-first}].
+
+[id="sandboxed-containers-1-2-bug-fixes"]
+== Bug fixes
+
+* Previously, using loop devices in a sandboxed container was not possible due to missing kernel modules. With this release, these kernel modules are included in the package. (link:https://issues.redhat.com/browse/KATA-1334[*KATA-1334*])
+
+* Previously, the `MachineConfigPool` (MCP) created by the Operator to track nodes with the `Kata` runtime installed was not automatically removed on deletion of the `KataConfig` custom resource (CR). With this release, the deletion of the `KataConfig` CR results in the removal of the `kata-oc` MCP. (link:https://issues.redhat.com/browse/KATA-1184[*KATA-1184*])
+
+* Previously, when you created a `kataConfigPoolSelector` field and changed it, the {sandboxed-containers-operator} did not apply the change. With this release, the Operator acts on changes to the `kataConfigPoolSelector` field in the CRD and adapts installations of the runtime on nodes accordingly. (link:https://issues.redhat.com/browse/KATA-1190[*KATA-1190*])
```

Comment:
Because this is the only instance of CRD, it should be spelled out.
```suggestion
* Previously, when you created a `kataConfigPoolSelector` field and changed it, the {sandboxed-containers-operator} did not apply the change. With this release, the Operator acts on changes to the `kataConfigPoolSelector` field in the custom resource definition and adapts installations of the runtime on nodes accordingly. (link:https://issues.redhat.com/browse/KATA-1190[*KATA-1190*])
```